src/toml.test.ts

```diff
@@ -115,4 +367,421 @@ foo = 'fizz'
     }
     expect(result).toEqual(expected)
   })
+
+  it('should read complex values by default', () => {
```

Comment:
Would it make sense to combine the tests for back and forth conversion, to save some duplicaiton?

Reply:
I would like to keep each test easily debug-able and make it do one thing. Having variables defined centrally or somehow combined tests would make it harder to see what a test does.